src/main/java/com/trailfinder/dao/EventDAOStub.java

```diff
@@ -15,13 +13,11 @@
 
 	@Override
 	public EventDTO getEvent(int eventId) {
-		// TODO Auto-generated method stub
```

Comment:
I know TODOs are frowned on when not addressed, but they do add value, and this is a good case.   A method that promises to getEvent and always returns null is frowned on as well.  If you're removing the TODO, I'd go one step further and give at least some implementation to this method.  Otherwise, the TODO, as annoying as it is, is a good reminder that there is work left TODO.  Good Git clients will remind you when you are committing code that has unanswered TODOs.  :)